docs/community/working-group.md

```diff
@@ -0,0 +1,189 @@
+# Working Groups
+
+[Working Groups (WGs)][wg] are collections of contributors with knowledge of specific components or
+technologies in the PowerShell domain.
+They are responsible for issue triage/acceptance, code reviews, and providing their expertise to
+others in issues, PRs, and RFC discussions.
+
+The list, description, and membership of the existing Working Groups is available
+[here](working-group-definitions.md).
+
+## Terms
+
+* **Contributor** is used interchangeably within this doc as anyone participating in issues or
+  contributing code/RFCs/etc, regardless of their status with the PowerShell project.
```

Comment:
Suggestion to include "documentation, tests, bug reports" before "etc." in order to be explicit about that work as valuable contributions.